tap.js

```diff
@@ -37,8 +59,9 @@
             this.el.addEventListener('touchend', this, false);
             this.el.addEventListener('touchcancel', this, false);
 
-        } else if (e.type === 'mousedown') {
+        } else if (e.type === 'mousedown' && this.whichbutton(e) === 1) {
```

Comment:
Even though event.which is non standard, it looks like a good enough fallback (with exception for old IE) https://developer.mozilla.org/en-US/docs/Web/API/MouseEvent/which